includes/class-wc-stripe-webhook-handler.php

```diff
@@ -1096,6 +1096,21 @@ public function process_payment_intent( $notification ) {
 
 					$charge->is_webhook_response = true;
 					$this->process_response( $charge, $order );
+
+					try {
+						/**
+						 * Fires after a webhook has been processed, but before we respond to Stripe.
+						 * This allows for custom processing of the webhook after it has been processed.
+						 *
+						 * @since 9.8.0
+						 *
+						 * @param string $webhook_type The type of webhook that was processed.
+						 * @param object $notification The webhook data sent from Stripe.
+						 */
+						do_action( 'wc_stripe_webhook_processed', (string) $notification->type, $notification );
+					} catch ( Exception $e ) {
+						WC_Stripe_Logger::error( 'Error in wc_stripe_webhook_processed action: ' . $e->getMessage() );
```

Comment:
We can now add the full error to the log context:
```suggestion
						WC_Stripe_Logger::error( 'Error in wc_stripe_webhook_processed action: ' . $e->getMessage(), [ 'error' => $e ] );
```